### PR TITLE
RESTWS-645 Added version to Module Resource default representation

### PR DESCRIPTION
**Pull request by suthagar23**
Now able to get version through Module Resource default representation

### CODE REVIEW

**Comment by suthagar23** [comment]
@dkayiwa @djazayeri  Can you merge this change?

**Comment by dkayiwa** [comment]
There should be a test for other properties. Can you add this also to the test?

**Comment by dkayiwa** [comment]
@suthagar23 did you see my comment?

**Comment by suthagar23** [comment]
@dkayiwa I have added some common test cases. Did you mean this?

**Comment by dkayiwa** [comment]
You should only test the changes you have made. If you would like to test other aspects, then you would need a new ticket and pull request.

**Comment by suthagar23** [comment]
@dkayiwa  Understood 👍 
I have modified the test cases for this ticket

File: omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/ModuleController1_8Test.java [hunk 1]
```diff
@@ -96,6 +96,14 @@ public void shouldIncludeAuthorToFullRepresentation() throws Exception {
 	}
 	
 	@Test
+	public void shouldIncludeVersionToDefaultRepresentation() throws Exception {
```
Comment (dkayiwa) [inline omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/ModuleController1_8Test.java]: Include version to? or Include version in?

Comment (suthagar23) [inline omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/ModuleController1_8Test.java]: I followed the method "shouldIncludeAuthorToFullRepresentation" :-)

**Pull request merged**